packages/node-fetch-server/src/node-fetch-server.ts

```diff
@@ -6,3 +6,7 @@ export {
   createRequest,
   sendResponse,
 } from './lib/request-listener.js';
+
+// hybrid magic
+export { FetchIncomingMessage } from './lib/fetch-incoming-message/index.js';
+export { createFetchServer } from './lib/create-fetch-server.js';
```

Comment:
Note: Ideally instead of creating two APIs we'd stick with a single API, `createRequestListenener`. Although I understand that you're exporting `createFetchServer` here under a different name so you can benchmark them both together.